src/campaign.js

```diff
@@ -226,13 +227,14 @@ function makeCampaignException(call, err) {
 
   if (err.statusCode && ctor && ctor.name == "HttpError") {
     var faultString = err.statusText;
-    var details = err.data;
+    var details = typeof err.data == 'object' ? JSON.stringify(err.data) : err.data;
```

Comment:
Why is this needed? I do not see anywhere where an object would be returned in err.data?

Reply:
it is object when you explicitly return json output as response from jssp API.